mesonbuild/modules/python.py

```diff
@@ -96,12 +113,17 @@ def __init__(self, python_holder, environment, kwargs):
         else:
             self.pkgdep = None
 
+            # 2. ... and will only try to find python
+            #    via SYSCONFIG as a fallback
             if DependencyMethods.SYSCONFIG in self.methods:
                 if mesonlib.is_windows():
                     self._find_libpy_windows(environment)
                 else:
                     self._find_libpy(python_holder, environment)
 
+                if self.is_found:
+                    mlog.debug('Found "python-{}" via SYSCONFIG module'.format(self.version))
+
         if self.is_found:
             mlog.log('Dependency', mlog.bold(self.name), 'found:', mlog.green('YES'))
```

Comment:
We should print here what mechanism was used to find the dependency, `pkgconfig` vs `sysconfig`.